pkg/catalog/catalog.go

```diff
@@ -15,11 +15,12 @@ type CatalogImage struct {
 	Namespace     string
 	AllTags       []string
 	NonHiddenTags []string
+
+	imageStreamRef imagev1.ImageStream
```

Comment:
This needs a comment explaining what it is for

Reply:
added